doc/attributes.md

```diff
@@ -67,3 +67,47 @@ message:
 
 When an attribute is used in Istio, its name is given along with a value. The value must be of the type declared by the corresponding descriptor. This
 type checking makes it possible for the Istio system to statically detect or prevent many configuration errors.
+
+# Standard Istio Attribute Vocabulary
+
+In a standard Istio deployment (using the Istio proxy), the system will produce the following set of attributes. This is considered the canonical attribute set for Istio.
+
+| Name | Type | Description | Kubernetes Example |
+|------|------|-------------|--------------------|
+| source.ip | ip_address | The IP address of the client that sent the request. | 10.0.0.117 |
+| source.port | int64 | The port on the client that sent the request. | 9284 |
+| source.name | string | The fully qualified name of the application that sent the request. | redis-master.namespace.deps.cluster.local |
+| source.uid | string | A unique identifier for the particular instance of the application that sent the request. | redis-master-2353460263-1ecey.namespace.pods.cluster.local |
+| source.namespace | string | The namespace of the sender of the request. | namespace.cluster.local |
```

Comment:
K8s has namespaces, but that doesn't mean the notion of a namespace is k8s specific. Namespace seems like a general enough name, but maybe we could come up with an alternate name describing the hierarchy?